src/pages/register-page/types.ts

```diff
@@ -1,8 +1,25 @@
-type InputFieldType = 'text' | 'email' | 'password' | 'date' | 'country-select';
+export type FormValues = {
```

Comment:
```suggestion
export type RegisterFormValues = {
```
у тебя ж будет еще форма логинки с другими полями